.github/workflows/cd.yml

```
@@ -20,11 +20,17 @@ jobs:
     - name: Checkout repo
       uses: actions/checkout@v2
     - name: Install SSH agent
-      uses: webfactory/ssh-agent@v0.4.0
+      uses: webfactory/ssh-agent@v0.4.1
       with:
         ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
     - name: Build relic
       run: make crypto/relic/build
+    - name: Docker login
```

My understanding is that, the `Docker push` step never works before, and we fixed it with this `Docker login` step? (because, otherwise, how did the `Docker push` push the image to gcr without this step?)

If so, better to explain the intention here.
```suggestion
    # provide credentials to github action, otherwise the `Docker push` step will fail
    # https://github.com/docker/login-action#google-container-registry-gcr
    - name: Docker login
```

---

As mentioned in the PR description, we were waiting on actually using this to build and push the image. I.e. We were never able to push to the registry before since we never logged in.

I personally think the Docker Login step is clear without any further clarification, as I don't think it's very ambiguous about what it's doing (logging in). However, happy to be explicit, so will add.